website/src/content/api-reference/commercetools-frontend-application-shell.mdx

```diff
@@ -73,6 +73,42 @@ export default EntryPoint;
 | `render`                         | `func`             |            (*)           | -       | The render function is called when the `<ApplicationShell>` is ready to render the actual application. This is the case when the required data (user, project) has been fetched and the application context has been initialized. It's **recommended to use the `children`** prop to benefit from a simpler setup.                                                                                                                                                                                                                                  |
 | `apolloClient`         | `ApolloClient`           |                       | -       | An optional instance of [ApolloClient](https://www.apollographql.com/docs/react/) to be used instead of the default one. This is usually the case when you need to configure the Apollo cache. See [`createApolloClient`](#createapolloclient).                                                                                                                                                                                                                                        |
 
+
+## ApplicationPageTitle
+
+<Info>
+
+This feature is available from version `21.15.0` onwards.
+
+</Info>
+
+This component can be used to overwrite the default document's `<title>`.
+
+### Usage
+
+We recommend to render this component in pages where there is a human-readable resource identifier, for example a Product name in the product details page.
+
+```js
+import { ApplicationPageTitle } from '@commercetools-frontend/application-shell';
+
+<ApplicationPageTitle additionalParts={['Red shoes']} />;
+// Red shoes - Products - my-shop - Merchant Center
+```
+
+The `<ApplicationPageTitle>` component can be rendered multiple times and the last one rendered will overwrite the previous ones.
+
+<Info>
+
+Please refer to the [Mapping guidelines](/development/human-readable-page-title#mapping-guidelines) to understand when you should overwrite the title and when not.
```

Comment:
Please refer to the [Mapping guidelines](/development/human-readable-page-title#mapping-guidelines) to understand when to overwrite the title and when not.